src/palace/manager/api/sirsidynix_authentication_provider.py

```diff
@@ -56,15 +56,19 @@ class SirsiDynixHorizonAuthSettings(BasicAuthProviderSettings):
         alias="CLIENT_ID",
     )
 
-    patron_status_block: bool = FormField(
+    patron_blocks_enforced: bool = FormField(
         True,
         form=ConfigurationFormItem(
-            label="Patron Status Block",
+            label="Enforce Patron ILS Blocks",
             description=(
-                "Block patrons from borrowing based on the status of the ILS's patron block field?"
+                "Block patrons from borrowing based on the status of the ILS's patron block field? "
```

Comment:
It would be nice to be really explicit here. I'd suggest something like:
> Block patrons from borrowing based on the `approved`, `hasMaxDaysWithFines`, `hasMaxFines`, `hasMaxLostItem`, `hasMaxOverdueDays`, `hasMaxItemsCheckedOut` fields from the ILS. (Note: expired accounts are always blocked).